### PR TITLE
PHPC-1975: Disable SKIPIF caching for skip_if_not_clean on PHP 8.1

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/PHPC-1975

### CODE REVIEW

File: tests/utils/skipif.php
Comment (alcaeus) [inline tests/utils/skipif.php:16]: Nit: this suggests that this method skips tests if there's no cache. disable_skipif_caching seems more appropriate, but I defer to your judgment.

**Review by alcaeus — APPROVED** [review]
LGTM - wording can stay if you don't like the alternative.

File: tests/utils/skipif.php
Comment (jmikola) [reply tests/utils/skipif.php:16]: That thought did occur to me when writing this. I agree that `disable_skipif_caching` is a much better name.

**Pull request merged**
